apps/webapp/src/app/splash/splash.tsx

```diff
@@ -2,17 +2,21 @@ import styled from '@emotion/styled';
 import { Layout } from '../ui-elements/layout';
 import { Link as RouterLink } from 'react-router-dom';
 import { Button } from '@mui/material';
+import { useDocumentTitle } from '../document-title/use-document-title';
 
 const StyledRouterLink = styled(RouterLink)`
   text-decoration: none;
 `;
 
-export const Splash = () => (
-  <Layout>
-    <div>
-      <StyledRouterLink to={'/meeting/add'}>
-        <Button>stwórz wydarzenie</Button>
-      </StyledRouterLink>
-    </div>
-  </Layout>
-);
+export const Splash = () => {
+  useDocumentTitle('Stwórz wydarzenie');
```

Comment:
może po prostu "cebulendarz"? bo kontekstowy tytuł ma sens jak jest kontekst, a to główna strona apki